scanner/indexer/indexer.go

```diff
@@ -15,7 +17,7 @@ type Indexer struct {
 
 func NewIndexer(ctx context.Context) (*Indexer, error) {
 	// TODO: Update the conn string to something configurable.
-	pool, err := postgres.Connect(ctx, "TODO", "libindex")
+	pool, err := postgres.Connect(ctx, "postgresql:///postgres?host=/var/run/postgresql", "libindex")
```

Comment:
curious, do you have something running with this?

Reply:
@RTann This is the default way to access PSQL in the same host (UNIX sockets). Usually, the default database is called "postgres". This is a reasonable guess to run this locally with minimal changes in the person's environment, but your mileage may vary.